news/2020-11-29-a-labour-of-love.md

```diff
@@ -0,0 +1,103 @@
+
+## Schedule
+
+- **3 months:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/115) until 2021-02-28. There's a lot of work going into this, so you might need the extra time.
+- **2 weeks:** Submissions are judged by separate mapper and musician judging panels.
+- **2 weeks:** Community vote is opened for top 5 overall submissions
+
+Once each of these are completed, the winners will be announced!
+
+## Prizes
+
+No such thing as a contest without prizes! Unlike most contests however, there's a few different way to earn prizes this time around.
+
+2 groups will handle judging: **mappers (including storyboarders!) and musicians.** They'll be expected to evaluate the full package — the remixed song + beatmap + storyboard/video combo. 
+
+Each group has its own specialty of course, so we'll be rewarding prizes to the all members of the highest scoring team from each panel.
+
+- **Mapper panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge
+
+![Mappers' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_mapperspick@2x.png)
+
+- **Musician panel**
+    - **Runner-up:** 3 months of osu!supporter 
+    - **Winner:**  6 months of osu!supporter + *Musicians' Pick* profile badge
+
+![Musicians' Pick profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_musicianspick@2x.png)
+
+The legend himself, *Kuba Oms*, will be part of this panel!
+
+That's not all though! When panel results are in, we'll combine them to form the overall top 5 entries. These will be open for a *community vote*, meaning you can pick your favorite entry!
+
+Once the vote is finalized, we'll combine panel scores with community vote scores to determine the overall winner (weighting for each category is undecided).
+
+This winner will be announced during a **live stream showcase of the finalists** with commentary from some of the judges!
+
+- **Overall prizes**
+    - **Runner-up:** 3 months of osu!supporter + *Overall runner-up* profile badge
+    - **Winner:**
+        - 3 months of osu!supporter
+        - *Overall winner* profile badge
+        - *Elite Mapper*, *Elite Storyboarder*, and *Elite Remixer* user titles
+        - Submission fast-tracked to Loved status (or Ranked if applicable!)
+
+![Overall runner-up profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallrunnerup@2x.png) ![Overall winner profile badge](/wiki/shared/news/2020-11-29-a-labour-of-love/alol_overallwinner@2x.png)
+
+Also in case one team wins both a specific panel and overall, the overall prizes will take priority!
+
+## Rules
+
+Follow these rules if you don't want your hard work to be disqualified:
```

Comment:
considering the whole process is public from the beginning and that it's a pretty big effort compared to other contests, I think you should notify teams if their submission would be disqualified (provided they submit early, I guess) so they have time to fix it